integration

```diff
@@ -939,6 +939,7 @@
   "sockless-coding/garo_wallbox",
   "sockless-coding/panasonic_cc",
   "SoftXperience/home-assistant-foxess-api",
+  "soloam/ha-fireflyiii-integration",
```

Comment:
<https://github.com/soloam/ha-fireflyiii-integration/issues/42>
@soloam with that, you want to hold of on this for now?